server/issue.go

```diff
@@ -576,15 +587,27 @@ func injectTeamAllowedValues(metaInfo *jira.CreateMetaInfo, teamIDList []TeamLis
 					continue
 				}
 
-				if schemaRaw["custom"] != "com.atlassian.jira.plugin.system.customfieldtypes:atlassian-team" {
+				if schemaRaw["custom"] != teamFieldSchema {
 					continue
 				}
 
-				fieldMap["allowedValues"] = allowedValues
-				issueType.Fields[key] = fieldMap
+				lowerKey := strings.ToLower(key)
```

Comment:
Why do we change to lowercase ? Are we also doing the same when looking up to avoid casing mismatchs?